docs/02-app/01-building-your-application/06-configuring/02-eslint.mdx

```diff
@@ -202,6 +202,21 @@ The `next/core-web-vitals` rule set is enabled when `next lint` is run for the f
 
 > The `next/core-web-vitals` entry point is automatically included for new applications built with [Create Next App](/docs/app/api-reference/create-next-app).
 
+### TypeScript
+
+In addition to the Next.js ESLint rules, `create-next-app --typescript` will also add [`@typescript-eslint`'s base recommended rules] to your config:
+
+```json filename=".eslintrc.json"
+{
+  "extends": [
+    "plugin:@typescript-eslint/recommended-type-checked",
+    "next/core-web-vitals"
+  ]
+}
+```
+
+See [typescript-eslint > Configs](https://typescript-eslint.io/linting/configs) for details on those recommended rules.
```

Comment:
I'm not super confident in these docs. Writing in the Next.js/Vercel style is very new to me 😄 seeking suggestions for how to improve.

These are also dependent on the strategy that ends up being taken for where to place the TypeScript preset.

Reply:
I'll just assume these are fine :)